hello-1/conanfile.py

```diff
@@ -1,18 +1,21 @@
-from conans import ConanFile
+from conans import ConanFile, tools
 
 class HelloConan(ConanFile):
     name = "hello"
     version = "0.1"
     exports = "*"
     generators = "qmake"
+    settings = "os", "compiler", "arch", "build_type"
```

Comment:
The settings are extremely important. All things that affect the final binary, i.e. if changing any of those values will produce a different binary, they have to be defined in the recipe.

The main use of not having settings would be a header-only library (or a package with only data), as the resulting package will always be the same, irrespective of the settings, just some header files